src/sentry/workflow_engine/handlers/condition/slow_condition_query_handlers.py

```diff
@@ -165,12 +231,20 @@ def get_rate_bulk(
             current_time -= comparison_interval
         start, end = self.get_query_window(end=current_time, duration=duration)
 
+        if filters:
+            conditions = []
+            for filter in filters:
+                snuba_condition = self.convert_filter_to_snuba_condition(filter)
+                if snuba_condition:
+                    conditions.append(snuba_condition)
+
         with self.disable_consistent_snuba_mode(duration):
             result = self.batch_query(
                 group_ids=group_ids,
                 start=start,
                 end=end,
                 environment_id=environment_id,
+                conditions=conditions,
```

Comment:
i start using the name `conditions` for the arguments after they've been converted to snuba conditions, before that they are `filters`. other naming suggestions would be appreciated